### ISSUE TITLE
Editing your profile: Ordered list is coded as <p>

### ISSUE THREAD

**Issue by XiuJuanXia** [mmiron-HC/WST-RSAMS#72]
URL: https://hcdev.openplus.ca/en/documentation/editing-your-profile
User Agent: Chrome, JAWS
Language: Both

Issue Description: Beneath "Language preferences", visually look like an ordered list, but it's coded as <p> tag.
![image](https://user-images.githubusercontent.com/88776404/129195527-21dde1ef-68b4-4922-b579-db74a89d9ce4.png)

Steps to reproduce: Move mouse to the text, right click mouse, and then select "Inspect" to view the source code.

Expected results: An ordered list should be coded using <ol>. Please fix on all other pages.

**Comment by echouin** [comment]
Modified the two item list using the WYSIWYG editor ordered list button in both EN & FR. Verified Source code and it appears fine now.

**Comment by hydegeoff** [comment]
Confirmed it is now in an order list, closed.
